tests/framework/PostProcessors/BasicStatistics/gold/basicStatisticsGeneral/PointSetPostProcTest_dump.xml

```diff
@@ -2,7 +2,6 @@
   <DataSet type="Static">
     <general>
       <inputs>x0,y0,z0</inputs>
-      <pointwise_meta>ProbabilityWeight</pointwise_meta>
```

Comment:
? why?

Reply:
Yes, why is a basic statistics test being modified?

Reply:
Ah, because it was an "output" test, and so basically not checked.